README.md

```diff
@@ -71,10 +63,42 @@ be good to go.
 Building the front-end
 ----------------------
 
-Run `grunt build` to regenerate the front-end logic and css files. For production, 
+Run `grunt build` to regenerate the front-end logic and css files. For production,
 run with `grunt build:prod` to get minified versions, and run `grunt build:prod:true` to enable
 auto building whenever a front-end file changes. See `Gruntfile.js` for details.
 
 **NOTE:** Our front-end is in transition from an older architecture using Requirejs to
 a new one using our build system. Check in our IRC channel (irc.mozilla.org#thimble) for
 the latest state of development.
+
+Bramble code editor
+-------------------
+
+In order to load Thimble, the bramble code editor to be statically served separately.
+The simplest way to do this is to:
+
+1) Clone the editor (outside your cloned Thimble source) and checkout to the `bramble` branch
+
+```
+git clone https://github.com/humphd/brackets.git --recursive && git checkout bramble
+```
+
+2) Install all dependencies (you will need to be in the brackets directory)
+
+```
+npm install && git submodule update --init
+```
+
+3) Host a server that can serve Bramble. Here are some examples:
```

Comment:
*Serve the root folder of the repo statically with a server. Thimble will look for it on localhost (port 8000 by default). Here are some examples:*